hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -0,0 +1,81 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_sysrst_ctrl
+  testpoints: [
+    {
+      name: chip_sw_sysrst_ctrl_combo_reset
+      desc: '''Verify that sysrst_ctrl issue reset on combo detection
+
+            - Initialize sysrst_ctrl with combo detect and output of combo detect as rst_req assertion.
+            - Wait for a delay greater than combo detect timer
+            - Check if there is a reset and if reset source is sysrst_ctrl
+            - Check if sysrst_ctrl.WKUP_STATUS.wakeup_sts is set
+            '''
+      features: ["SYSRST_CTRL.COMBO_DETECT"]
+      stage: V2
```

Comment:
Is this a new test? If so, we should make this V3 so that we do not regress our current verification stage. Same applies to the tests below.

Reply:
Updated to V3